examples/densenet_pytorch/config.properties

```diff
@@ -0,0 +1,5 @@
+vmargs=-Xmx128m -XX:-UseLargePages -XX:+UseG1GC -XX:MaxMetaspaceSize=32M -XX:MaxDirectMemorySize=10m -XX:+ExitOnOutOfMemoryError
```

Comment:
This file is not required. You could just use "pip install mxnet-model-server" and "mxnet-model-server --start --models ..." . You can rely on default configuration.